src/lib.rs

```diff
@@ -1,4 +1,170 @@
-//! A pure-rust TDS implementation for Microsoft SQL Server (>=2008)
+//! An asynchronous, runtime-independent, pure-rust TDS implementation for
```

Comment:
I don't know if that's in scope for these docs to explain this, but I don't know what TDS stands for.

Reply:
Yeah, you kind of need to understand the Microsoft terminology. Maybe I'll extend this to Tabular Data Stream (TDS).